README.md

```diff
@@ -305,6 +284,44 @@ to Miri failing to detect cases of undefined behavior in a program.
   `-Zmiri-disable-validation` is set.
 * `-Zmiri-ignore-leaks` disables the memory leak checker, and also allows some
   remaining threads to exist when the main thread exits.
+* `-Zmiri-seed=<hex>` configures the seed of the RNG that Miri uses to resolve
+  non-determinism. This RNG is used to pick base addresses for allocations. When
+  isolation is enabled (the default), this is also used to emulate system
+  entropy. The default seed is 0. You can increase test coverage by running Miri
+  multiple times with different seeds.
+  **NOTE**: This entropy is not good enough for cryptographic use! Do not
+  generate secret keys in Miri or perform other kinds of cryptographic
+  operations that rely on proper random numbers.
+* `-Zmiri-strict-provenance` enables [strict
```

Comment:
Is this common at present, seems experimental. Once it leaves experimental mode, we should probably switch to it by default and add a disable flag. Though seems ok here for visibility

Reply:
I don't think strict-provenance should be the default, see https://github.com/rust-lang/miri/issues/2133.